### DISCUSSION TITLE
How to uninstall a Babel plugin/I'm not sure what I'm doing

### DISCUSSION THREAD

**Discussion by afangster** [babel/babel#15714] [General]
I am new to using React Native, Babel, Expo, etc. I was following a tutorial on using React Native and I followed the instructions to install the Babel plugin:
```
npm install --save-dev babel-plugin-react-native-web. 
```
I am now trying to start from scratch and want to uninstall this plugin so that I can follow Expo's tutorial (https://docs.expo.dev/workflow/web/) and use webpack.config.js instead of having a babel.config.js, but I am not sure how to uninstall the plugin from Windows. How should I do this?

Edit: I have tried to uninstall it with:
```
npm uninstall babel-plugin-react-native-web
npm uninstall -D babel-plugin-react-native-web
```
It doesn't show up under 'npm list' but I still have a babel.config.js file when the project is created and babel still shows up under my devDependencies. I am honestly confused and am not sure what I am doing. Is this supposed to be the case? Other tutorials I have looked at have a webpack.config.js file that I don't have.

I also have a bunch of warnings now when I run npx expo install react-dom react-native-web @expo/webpack-config that I didn't have before. 
```
warning "expo > babel-preset-expo > @babel/preset-env > babel-plugin-polyfill-corejs2@0.4.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "expo > babel-preset-expo > @babel/preset-env > babel-plugin-polyfill-corejs3@0.8.1" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "expo > babel-preset-expo > @babel/preset-env > babel-plugin-polyfill-regenerator@0.5.0" has unmet peer dependency "@babel/core@^7.0.0-0".
```

**Comment by nicolo-ribaudo** [comment]
The easiest way to uninstall dependencies is to delete them from your `package.json` file, and then run `npm install`.
However, I believe that Expo _needs_ `@babel/core`, so you need to keep at least that in your devDependencies. There _might_ be a way to use it without Babel, but you should ask directly to them 🙂
